libs/cart/state/src/selectors/cart-item-entities/cart-item-entities.selectors.ts

```diff
@@ -21,6 +21,30 @@ export interface DaffCartItemEntitiesMemoizedSelectors<T extends DaffStatefulCar
 	selectIsCartItemOutOfStock: MemoizedSelectorWithProps<object, object, boolean>;
 	selectCartItemMutating: MemoizedSelector<object, boolean>;
 	selectCartItemState: MemoizedSelectorWithProps<object, object, DaffCartItemStateEnum>;
+  /**
```

Comment:
I think we should also mention in these comments all the stuff about including category/product discounts and excluding cart discounts, or whatever they should be

Reply:
changed